utils/id.go

```diff
@@ -66,17 +73,91 @@ func LocalNodeID() (string, error) {
 	return fmt.Sprintf("%s%s", NodePrefix, HashedID(hostname)[:8]), nil
 }
 
-var b62Index = newB62Index()
-var b62Chars = []byte(shortuuid.DefaultAlphabet)
+var b57Index = newB57Index()
```

Comment:
why do we move from 62 to 57?

Reply:
yeah, this variable has been mislabeled for a while. the author of the shortuuid lib made up his own encoding and we're sort of stuck with it now.